src/client/components/controls/clickable-edit.js

```diff
@@ -1,11 +1,11 @@
 import React, { useState } from 'react';
 import {
-  Button, Form, Row, Col,
+  Button, Form, Row, Col, Spinner,
 } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 import { MdCancel, MdSave, MdEdit } from 'react-icons/md';
 
-const ClickableEdit = ({ text, setText }) => {
+const ClickableEdit = ({ text, setText, loading }) => {
```

Comment:
This is a rudimentary async mode - it just uses a `loading` prop to indicate that an operation is in progress. Its most serious bug is that if the edit fails, it currently does not receive that information properly. GH-53 has been opened to fix this.